### PR TITLE
Added some steps how to contribute on project

### PR DESCRIPTION
**Pull request by Atul8766**
Add all the steps that is required during setup the project on our local sytem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,11 +36,52 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
 
 To contribute new features and changes to the website, you would want to run the app locally. Please follow these steps:
 
+
+## How to setup the project in their local machine
```
Comment (lucavallin) [inline README.md:40]: Use "How to setup the project locally" instead

File: README.md [hunk 2]
```diff
@@ -36,11 +36,52 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
...
+
 1. Clone the project locally. Make sure you have a recent version of Node.js installed on your computer.
 
-2. You can use the included [generated.json](generated.json) as dummy data or you can run `npm run prebuild` to fetch the latest data from GitHub yourself: for this, you will need to set the `GH_PERSONAL_ACCESS_TOKEN` environment variable to a valid GitHub Personal Access Token (PAT). Notice: repositories not maching the criteria listed above (see rules in [generated.json](generated.json)) are automatically removed from [firstissue.json](firstissue.json) when the [generated.json](generated.json) script runs.
+2. Fork the repository
+Forking the repository means creating a copy of the original repository in your Github account. To do this, navigate to the repository you want to contribute to and click on the “Fork” button in the top-right corner.
```
Comment (lucavallin) [inline README.md:45]: This is not necessary for just running the project locally. Also, forking should be done before step 1, therefore first fork, then clone.

File: README.md [hunk 3]
```diff
@@ -36,11 +36,52 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
...
+
+3. Clone the repository
```
Comment (lucavallin) [inline README.md:47]: Already in step 1.

File: README.md [hunk 4]
```diff
@@ -36,11 +36,52 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
...
+Pushing changes means uploading the changes to your forked repository on Github. To push changes, type the following command in your terminal.
+
+```bash
+$ git push origin <branch-name>
+```
+
+8. Submit a pull request
+A pull request is a request to the original repository’s maintainer to merge your changes into the main branch. To submit a pull request, navigate to your forked repository on Github, switch to the branch you just pushed, and click on the “New pull request” button. Fill in a descriptive title and comments explaining the changes you made and why they are necessary.
+
+9. You can use the included [generated.json](generated.json) as dummy data or you can run `npm run prebuild` to fetch the latest data from GitHub yourself: for this, you will need to set the `GH_PERSONAL_ACCESS_TOKEN` environment variable to a valid GitHub Personal Access Token (PAT). Notice: repositories not maching the criteria listed above (see rules in [generated.json](generated.json)) are automatically removed from [firstissue.json](firstissue.json) when the [generated.json](generated.json) script runs.
```
Comment (lucavallin) [inline README.md:82]: Committing, pushing and opening a pull request are not requirements for making changes locally.

**Comment by lucavallin** [comment]
Thank you for your contribution! Overall, I wouldn't use our README to explain how git works - I would condense points 1-8 into a very short paragraph. Also, the points regarding committing, pushing and opening a PR come, in the list, before starting the project locally - these steps should be taken afterwards instead, should a person have something to contribute to the project. Point 9 and 10 instead are specific to this project (they explain how it works) and we could expand on those.

**Comment by Atul8766** [comment]
Okay thanks for your response I will make the correction and Request a PR again.

**Comment by Atul8766** [comment]
@lucavallin I did some changes according to your requirements let me know is this is right. feel free to give feedback

File: README.md [hunk 5]
```diff
@@ -36,11 +36,24 @@ First Issue is a static website that uses Next.js, React and Typescript. The dat
 
 To contribute new features and changes to the website, you would want to run the app locally. Please follow these steps:
 
-1. Clone the project locally. Make sure you have a recent version of Node.js installed on your computer.
 
-2. You can use the included [generated.json](generated.json) as dummy data or you can run `npm run prebuild` to fetch the latest data from GitHub yourself: for this, you will need to set the `GH_PERSONAL_ACCESS_TOKEN` environment variable to a valid GitHub Personal Access Token (PAT). Notice: repositories not maching the criteria listed above (see rules in [generated.json](generated.json)) are automatically removed from [firstissue.json](firstissue.json) when the [generated.json](generated.json) script runs.
+## How to setup the project locally
+1. Fork the repository. To do this, navigate to the repository for your contribution and click on the “Fork” button in the top-right corner.
```
Comment (lucavallin) [inline README.md:41]: Nice, thanks for the update! Two things:

- Step 3 is missing
- I would merge points 1, 2,4 along the line of "Fork and Clone the repository. Then create a new branch. You can do so by: blablabla and then `git clone ...` and then `git checkout ...`"

**Review by lucavallin — APPROVED** [review]
Awesome, thank you!

**Comment by Atul8766** [comment]
@lucavallin Can you please merge the changes now and close the issue?

**Comment by Atul8766** [comment]
@lucavallin Can you please merge the changes now and close the issue?

**Pull request merged**